Makefile

```diff
@@ -16,12 +16,14 @@
 VERSION?=dev-$(shell date +%FT%T%z)
 KS_BIN?=ks
 
-APIMACHINERY_VER := $(shell dep status | grep k8s.io/apimachinery | awk '{print $$3}')
+APIMACHINERY_VER := $(shell grep -B1 k8s.io/apimachinery Gopkg.lock | head -n1 | cut -d'"' -f2)
 REVISION=$(shell git rev-parse HEAD)
+GIT_TAG=$(shell git describe --tags)
```

Comment:
Would it make sense to only set `GIT_TAG` if is needed?

Reply:
so in Makefiles variables set with = are lazily evaluated ... so it really doesn't hurt anything to make it available here as it won't run `git describe` unless you are building a docker image. but i don't mind moving this into the target if that seems cleaner?